File: ODOT.ARMS.Web/Entities/ControllingBoard.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ODOT.ARMS.Web.Entities
{
    public class ControllingBoard
    {
        public Guid ControllingBoardId { get; set; }
        public Guid ProjectId { get; set; }
        public string ControllingBoardNumber { get; set; }
        public int ControllingBoardType { get; set; }
        public int ControllingBoardStatus { get; set; }
        public string PublicCommentText { get; set; }
        public string UserId { get; set; }
        public DateTime EntryDate { get; set; }
        public string ActiveInd { get; set; }
        public DateTime ControllingBoardDate { get; set; }
        public DateTime SubmissionDate { get; set; }
        public int ControllingBoardCategory { get; set; }
    }
}

File: ODOT.ARMS.Web/ClientApp/src/app/project/models/controllingBoard.ts
import { EventUpload } from './events';

export interface IControllingBoard {
  controllingBoardId: string;
  projectId: string;
  controllingBoardNumber: string;
  controllingBoardType: number;
  controllingBoardStatus: number;
  publicCommentText: string;
  userId: string;
  entryDate: Date;
  activeInd: string;
  controllingBoardDate: Date;
  submissionDate: Date;
  controllingBoardCategory: number;
  document: string;
  eventUploadForDD: Array<EventUpload>;
  files: Array<File>;
}
export class ControllingBoardRaw implements IControllingBoard {
  controllingBoardId: string;
  projectId: string;
  controllingBoardNumber: string;
  controllingBoardType: number;
  controllingBoardStatus: number;
  publicCommentText: string;
  userId: string;
  entryDate: Date;
  activeInd: string;
  document: string;
  controllingBoardDate: Date;
  submissionDate: Date;
  controllingBoardCategory: number;
  eventUploadForDD: Array<EventUpload>;
  files: Array<File>;
}


